azure_arc_servers_jumpstart/docs/arc_policies_mma.md

```diff
@@ -0,0 +1,121 @@
+
+    ![](../img/vm_policies/01.png)
+
+    ![](../img/vm_policies/02.png)
+
+  
+* [Install or update Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest). Azure CLI should be running version 2.7** or later. Use ```az --version``` to check your current installed version.
+
+* Create Azure Service Principal (SP)   
+
+    To connect a VM or bare-metal server to Azure Arc, Azure Service Principal assigned with the "Contributor" role is required. To create it, login to your Azure account run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/)).
+
+  ```bash
+    az login
+    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor
+    ```
+    For example:
+    ```az ad sp create-for-rbac -n "http://AzureArcServers" --role contributor```
+    Output should look like this:
+    ```
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcServers",
+    "name": "http://AzureArcServers",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+    
+**Note**: It is optional but highly recommended to scope the SP to a specific [Azure subscription and Resource Group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest).
+
+* You will also need to have a Log Analytics Workspace deployed. You can automate the deployment by editing the ARM template [parameters file](../policies/arm/log_analytics-template.parameters.json), provide a name and location for your workspace
+
+    ![](../img/vm_policies/03.png)
+
+To deploy the ARM template, navigate to the [deployment folder](../policies/arm) and run the below command:
+
+  ```bash
+    az deployment group create --resource-group <Name of the Azure Resource Group> \
+    --template-file <The *log_analytics-template.json* template file location> \
+    --parameters <The *log_analytics-template.parameters.json* template file location>
+  ```
+
+# Azure Policies on Azure Arc connected machines
+
+* Now that you have all the requirements set, you can assign policies to our Arc connected machines. Edit the [parameters file](../policies/arm/policy.json) to provide your subscription ID as well as the Log Analytics Workspace. 
+
+    ![](../img/vm_policies/04.png)
+
+To start the deployment, use the below command: 
+
+```bash
+az policy assignment create --name 'Enable Azure Monitor for VMs' \
+--scope '/subscriptions/<Your subscription ID>/resourceGroups/<Name of the Azure Resource Group>' \
+--policy-set-definition '55f3eceb-5573-4f18-9695-226972c6d74a' \
+-p <The *policy.json* template file location> \
+--assign-identity --location <Azure Region>
+```
+
+The flag *policy-set-definition* points to the initiative "Enable Azure Monitor" definition ID. 
+
+* Once you have assigned the initiative, you will see that it will be evaluated (it may take 30 minutes to run the first scan) and show that the server on GCP is not compliant.
```

Comment:
```suggestion
* Once you have assigned the initiative, you will see that it will be evaluated (it may take ~30 minutes to run the first scan) and show that the server on GCP is not compliant.
```

Reply:
I think it needs to have a better explanation on the flow. See here:
https://github.com/likamrat/azure_arc/blob/master/azure_arc_k8s_jumpstart/docs/aks_policy.md